src/components/volunteer/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from './VolunteerApplicantListItem';
+export * from './VolunteerApplicantListItem';
```

Comment:
new duplication here?

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/617/commits/6ea15dda586b5bd947e8b7cd4a2facb746f89026

Reply:
updating the import in the other file was the correct solution